spec/rails_helper.rb

```diff
@@ -6,26 +6,17 @@
 require "spec_helper"
 require "rspec/rails"
 require "webmock/rspec"
-# Add additional requires below this line. Rails is not loaded until this point!
+require "vcr"
+require "json-schema"
```

Comment:
agree, i'll make this change as well -- just let me know when you're done reviewing.